### ISSUE TITLE
cant build flutter_scene

### ISSUE THREAD

**Issue by wangcker** [bdero/flutter_scene#38]
hello,
I try to build scene lib,but it throws:

```
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/code/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'
```

flutter doctor -v
```
[✓] Flutter (Channel master, 3.27.0-1.0.pre.352, on macOS 14.6.1 23G93 darwin-arm64, locale zh-Hans-CN)
    • Flutter version 3.27.0-1.0.pre.352 on channel master at /Users/wangcker/opt/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 8591d0c16a (13 hours ago), 2024-11-04 20:46:29 +0800
    • Engine revision 05cb5d7f79
    • Dart version 3.7.0 (build 3.7.0-95.0.dev)
    • DevTools version 2.40.1

[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    • Android SDK at /Users/wangcker/Library/Android/sdk
    • Platform android-35, build-tools 34.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 16.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 16B40
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2024.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)

[✓] IntelliJ IDEA Ultimate Edition (version 2023.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA 2.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.95.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.100.0

[✓] Connected device (5 available)
    • 2201122C (mobile)               • b5125962                                 • android-arm64  • Android 14 (API 34)
    • wangcker的iPhone (mobile)        • dbdeac590c9d9faa0c3068044290a3f9ef6a3b19 • ios            • iOS 15.3.1 19D52
    • macOS (desktop)                 • macos                                    • darwin-arm64   • macOS 14.6.1 23G93 darwin-arm64
    • Mac Designed for iPad (desktop) • mac-designed-for-ipad                    • darwin         • macOS 14.6.1 23G93 darwin-arm64
    • Chrome (web)                    • chrome                                   • web-javascript • Google Chrome 130.0.6723.92

[✓] Network resources
    • All expected network resources are available.

• No issues found!
```

**Comment by kwill39** [comment]
Hey @wangcker, have you installed CMake prior to trying to run Flutter scene? That's a common issue that trips up a lot of people. You have to install CMake before running the model converter.

**Comment by wangcker** [comment]
@kwill39 
cmake --version

```
cmake version 3.28.1

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

**Comment by kwill39** [comment]
@wangcker
Perfect 

Good to see that's not the issue. Can you paste in the command that you're using and a picture of the file structure of your root project? Feel free to block out / replace any personal or sensitive info

**Comment by wangcker** [comment]
> @wangcker Perfect
> 
> Good to see that's not the issue. Can you paste in the command that you're using and a picture of the file structure of your root project? Feel free to block out / replace any personal or sensitive info

just cloned from flutter_scene from github
it likely occur   at    FetchContent_MakeAvailable(flatbuffers)  in importer/CMakeLists.txt

**Comment by wangcker** [comment]
my project ref the flutter_scene in pubspec.yaml by this:
  flutter_scene:
    path: ../flutter_scene

**Comment by kwill39** [comment]
Oh I see what you're trying to do now. I haven't tried using a local copy of Flutter scene and referring to it directly before.

bdero would probably be more familiar with building it locally, and I'm sure they'll see this ticket fairly soon and be able to comment with better advice.

**Comment by kwill39** [comment]
@wangcker If you do find the issue before then, definitely comment here if you don't mind! I'd love to know what the issue was myself.

**Comment by wangcker** [comment]
all the error info:

```
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
Building native assets for package:flutter_scene_importer failed.
file:///Users/**/flutter_scene/importer/hook/build.dart returned with exit code: 255.
To reproduce run:
/Users/wangcker/opt/flutter/bin/dart --packages=/Users/**/my_3d_app/.dart_tool/package_config.json /Users/**/my_3d_app/.dart_tool/native_assets_builder/ad5635314aae4ae03d576269d98dd699/hook.dill --config=/Users/**/my_3d_app/.dart_tool/native_assets_builder/ad5635314aae4ae03d576269d98dd699/config.json
stderr:
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

stdout:
Running cmake gen step...
CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!



Unhandled exception:
Exception: Unable to find build executable flatc! Tried the following locations: [file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Release/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Release/flatc.exe, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Debug/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Debug/flatc.exe, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/flatc.exe]
#0      findBuiltExecutable (package:flutter_scene_importer/offline_import.dart:27:5)
#1      generateImporterFlatbufferDart (package:flutter_scene_importer/offline_import.dart:42:17)
#2      main.<anonymous closure> (file:///Users/**/flutter_scene/hook/build.dart:8:5)
#3      build (package:native_assets_cli/src/api/build.dart:93:16)
#4      main (file:///Users/**/flutter_scene/hook/build.dart:7:9)
#5      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#6      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
Building native assets for package:flutter_scene failed.
file:///Users/**/flutter_scene/hook/build.dart returned with exit code: 255.
To reproduce run:
/Users/wangcker/opt/flutter/bin/dart --packages=/Users/**/my_3d_app/.dart_tool/package_config.json /Users/**/my_3d_app/.dart_tool/native_assets_builder/0d50bf83c90021c6c9d86d23ebf805fb/hook.dill --config=/Users/**/my_3d_app/.dart_tool/native_assets_builder/0d50bf83c90021c6c9d86d23ebf805fb/config.json
stderr:
Unhandled exception:
Exception: Unable to find build executable flatc! Tried the following locations: [file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Release/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Release/flatc.exe, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Debug/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/Debug/flatc.exe, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/flatc, file:///Users/**/flutter_scene/importer/build/_deps/flatbuffers-build/flatc.exe]
#0      findBuiltExecutable (package:flutter_scene_importer/offline_import.dart:27:5)
#1      generateImporterFlatbufferDart (package:flutter_scene_importer/offline_import.dart:42:17)
#2      main.<anonymous closure> (file:///Users/**/flutter_scene/hook/build.dart:8:5)
#3      build (package:native_assets_cli/src/api/build.dart:93:16)
#4      main (file:///Users/**/flutter_scene/hook/build.dart:7:9)
#5      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#6      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

stdout:


Unhandled exception:
Exception: Failed to run flutter_scene_importer:import command in build hook (exit code 255):
STDERR: Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
Building native assets for package:flutter_scene_importer failed.
file:///Users/**/flutter_scene/importer/hook/build.dart returned with exit code: 255.
To reproduce run:
/Users/wangcker/opt/flutter/bin/cache/dart-sdk/bin/dart --packages=/Users/**/my_3d_app/.dart_tool/package_config.json /Users/**/my_3d_app/.dart_tool/native_assets_builder/eb82b79aba38faeae02ae065523f1a96/hook.dill --config=/Users/**/my_3d_app/.dart_tool/native_assets_builder/eb82b79aba38faeae02ae065523f1a96/config.json
stderr:
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

stdout:
Running cmake gen step...
CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!



Error: Compiling native assets failed.

STDOUT:
#0      buildModels (package:flutter_scene_importer/build_hooks.dart:46:7)
#1      main.<anonymous closure> (file:///Users/**/my_3d_app/hook/build.dart:6:5)
#2      build (package:native_assets_cli/src/api/build.dart:93:16)
#3      main (file:///Users/**/my_3d_app/hook/build.dart:5:3)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#5      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
Building native assets for package:my_3d_app failed.
file:///Users/**/my_3d_app/hook/build.dart returned with exit code: 255.
To reproduce run:
/Users/wangcker/opt/flutter/bin/dart --packages=/Users/**/my_3d_app/.dart_tool/package_config.json /Users/**/my_3d_app/.dart_tool/native_assets_builder/79f989ad58edcd9ea7a456ca4fe90521/hook.dill --config=/Users/**/my_3d_app/.dart_tool/native_assets_builder/79f989ad58edcd9ea7a456ca4fe90521/config.json
stderr:
Unhandled exception:
Exception: Failed to run flutter_scene_importer:import command in build hook (exit code 255):
STDERR: Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
Building native assets for package:flutter_scene_importer failed.
file:///Users/**/flutter_scene/importer/hook/build.dart returned with exit code: 255.
To reproduce run:
/Users/wangcker/opt/flutter/bin/cache/dart-sdk/bin/dart --packages=/Users/**/my_3d_app/.dart_tool/package_config.json /Users/**/my_3d_app/.dart_tool/native_assets_builder/eb82b79aba38faeae02ae065523f1a96/hook.dill --config=/Users/**/my_3d_app/.dart_tool/native_assets_builder/eb82b79aba38faeae02ae065523f1a96/config.json
stderr:
Unhandled exception:
Exception: CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!

#0      main.<anonymous closure> (file:///Users/**/flutter_scene/importer/hook/build.dart:37:7)
#1      build (package:native_assets_cli/src/api/build.dart:93:16)
#2      main (file:///Users/**/flutter_scene/importer/hook/build.dart:7:9)
#3      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#4      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

stdout:
Running cmake gen step...
CMake generate step failed (exit code 1):
STDERR: [ 11%] Performing update step for 'flatbuffers-populate'
CMake Error at /Users/**/flutter_scene/importer/build/_deps/flatbuffers-subbuild/flatbuffers-populate-prefix/tmp/flatbuffers-populate-gitupdate.cmake:34 (message):
  Failed to get the hash for HEAD:

  fatal: not a git repository: '.git'



make[2]: *** [flatbuffers-populate-prefix/src/flatbuffers-populate-stamp/flatbuffers-populate-update] Error 1
make[1]: *** [CMakeFiles/flatbuffers-populate.dir/all] Error 2
make: *** [all] Error 2

CMake Error at /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1679 (message):
  Build step for flatbuffers failed: 2
Call Stack (most recent call first):
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819:EVAL:2 (__FetchContent_directPopulate)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:1819 (cmake_language)
  /opt/homebrew/Cellar/cmake/3.28.1/share/cmake/Modules/FetchContent.cmake:2033 (FetchContent_Populate)
  CMakeLists.txt:27 (FetchContent_MakeAvailable)



STDOUT: -- Configuring incomplete, errors occurred!



Error: Compiling native assets failed.

STDOUT:
#0      buildModels (package:flutter_scene_importer/build_hooks.dart:46:7)
#1      main.<anonymous closure> (file:///Users/**/my_3d_app/hook/build.dart:6:5)
#2      build (package:native_assets_cli/src/api/build.dart:93:16)
#3      main (file:///Users/**/my_3d_app/hook/build.dart:5:3)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:33)
#5      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)

stdout:


Building (dry run) native assets failed. See the logs for more details.

```

**Comment by bdero** [comment]
This is resolved by #43.
